zebra/zebra_fpm.c

```diff
@@ -1006,6 +1005,8 @@ static int zfpm_build_route_updates(void)
 					zfpm_g->stats.route_adds++;
 				else
 					zfpm_g->stats.route_dels++;
+			} else {
+				zlog_debug("zfpm_build_route_updates, data_len is 0.");
```

Comment:
no open debugs, please - try using zfpm_debug() instead?

please don't type out the name of the api - use `__func__` instead

Reply:
Hi,

At first, I use `zlog_debug("%s, data_len is 0.", __FUNC__);`.
But I got crash with log: `zlog ....`, so I will try your advertise, use `zlog_debug("%s, data_len is 0.", __func__);` to test.